### PR TITLE
Move properties as part of spring boot 2 migration

### PR DESCRIPTION
**Pull request by eddumelendez**
Fixes gh-5

### CODE REVIEW

**Comment by eddumelendez** [comment]
I think an integration test wouldn't be a good idea since if the value is replaced in `src/integrationTest/resources/application.properties` the check will fail. Due to this validation is kind of production ready for us I would suggest to add another point of validation for this. WDYT?

**Comment by yshewchuk** [comment]
I think maybe you misunderstood - we rely on the health check endpoint being available. In the previous change (upgrading to spring boot 2) we caused an issue with the configuration that could have potentially caused the health endpoint to no longer work. I'm suggesting that we write an integration test that sends a request to the health check endpoint ("/health") and verifies that it returns a valid result. This way, if someone accidentally removes the actuator configuration or dependency we'd immediately be aware that the health check endpoint no longer works. Does that sound agreeable?

**Comment by christophe-f** [comment]
I would leave the default Spring Boot settings and let everyone be able to customize on their needs.

If you deploy on a Cloud Foundry platform there is also the second set of Actuator endpoints automatically enabled (https://content.pivotal.io/blog/using-spring-boot-actuator-integrations-with-pivotal-cloud-foundry-111)

**Comment by eddumelendez** [comment]
>  This way, if someone accidentally removes the actuator configuration or dependency we'd immediately be aware that the health check endpoint no longer works. Does that sound agreeable?

@yshewchuk It is sounds good to me. I will add the check later today.

@christophe-f since this project is opinionated I understand one of the features is expose the `health` endpoint but of course as you said this can be customized.

File: src/integrationTest/java/com/scotiabank/accelerator/initializer/controller/AcceleratorHealthTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = WebEnvironment.MOCK)
+@AutoConfigureMockMvc
+public class AcceleratorHealthTest {
+
+	@Autowired
+	private MockMvc mvc;
+
+	@Test
```
Comment (yshewchuk) [inline src/integrationTest/java/com/scotiabank/accelerator/initializer/controller/AcceleratorHealthTest.java]: Adding `@SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert") // uses MockMVC assertions` should address codacy issue. This is the same approach taken for the other integration tests.

**Comment by yshewchuk** [comment]
Once the codacy fix is in I'll approve and merge

**Pull request merged**